packages/react-native/src/components/Fabric/Video.tsx

```diff
@@ -0,0 +1,31 @@
+import 'react-native-get-random-values';
+import React, { useState } from 'react';
+import type * as SignalWire from '@signalwire/js';
+import { Fabric } from '@signalwire-community/react';
+
+import { registerGlobals } from 'react-native-webrtc';
+import RemoteStream from '../RemoteStream';
+registerGlobals();
+
+interface IVideoNativeProps extends Fabric.IVideoProps {
+  style: any;
+}
+
+export function Video({ style, onRoomReady, ...props }: IVideoNativeProps) {
+  const [roomSession, setRoomSession] =
+    useState<SignalWire.Video.RoomSession | null>(null);
```

Comment:
just a note: this works for now, but the type will not be `RoomSession` anymore, eventually. But let's keep it like this for now.